demos/device_defender_for_aws/defender_demo.c

```diff
@@ -286,7 +286,7 @@ static bool validateDefenderResponse( const char * defenderResponse,
         {
             LogError( ( "reportId key not found in the response from the"
                         "AWS IoT Device Defender Service: %.*s.",
-                        ( int ) defenderResponseLength,
```

Comment:
The compiler does expect a signed integer for the length which is why that cast exists

Reply:
Ah, you are correct. Windows says unsigned but spec says int. Will add those back